vars/buildPlugin.groovy

```diff
@@ -147,26 +147,30 @@ def call(Map params = [:]) {
                             }
 
                             if (first) {
-                                referenceJobName = env.JOB_NAME.substring(0, env.JOB_NAME.lastIndexOf("/") + 1) + "master"
-                                echo "Recording static analysis results on '${stageIdentifier}' using reference job '${referenceJobName}'"
+                                folders = env.JOB_NAME.split("/")
+                                if (folders.length > 1) {
+                                    discoverGitReferenceBuild(scm: folders[1])
```

Comment:
isn't this determined automatically?

Reply:
Since we use 2 SCMs in our pipeline (shared library and plugin) I need to select the repository manually. I found no automatic way to decide which one to use.